catalog/cataloger_retention_test.go

```diff
@@ -653,7 +655,7 @@ func TestCataloger_MarkObjectsForDeletion(t *testing.T) {
 		t.Errorf("expected 1 object marked for deletion, got %v", count)
 	}
 
-	conn, err := db.ConnectDB("pgx", c.DbConnURI)
+	conn, err := db.ConnectDB(params.Database{URI: c.DbConnURI})
```

Comment:
`"pgx"`?

Reply:
made it the default on connect - I'm thinking if there is a reason why we would ever want to configure it at this level.
what do you think?

Reply:
I'd expect us to configure it to whatever is useful.  I mean, AFAICT the two main Golang Postgres drivers are pg, pq and pgx...  I don't understand the details well enough though.  If nobody will ever use a driver other than pgx (for instance if the other two won't work for what we need) then obviously an unchangeable value is good.
E.g. if you can select the driver (rather than just the protocol) from the URL then it's obviously good enough.  BTW, this applies to _all_ connections not just this one :-/